configure.ac

```diff
@@ -420,7 +420,23 @@ AM_CONDITIONAL([libao_support], [test $with_libao = "yes"])
 AC_SUBST([LIBAO_CFLAGS])
 AC_SUBST([LIBAO_LIBS])
 AS_IF([test $with_libao = "yes"], [audio_methods="${audio_methods} libao"])
-
+# check for pipewire support
+AC_ARG_WITH([pipewire],
+	[AS_HELP_STRING([--with-pipewire], [include pipewire support])],
+	[],
+	[with_pipewire=check])
+AS_IF([test $with_pipewire != "no"],
+	[PKG_CHECK_MODULES([pipewire], [libpipewire-0.3],
+		[with_pipewire=yes
+		AS_IF([test -z "$default_audio_method"],
+			[default_audio_method=pipewire])
```

Comment:
We don't want this precisely :)

Reply:
this is how pulse and alsa declared themselves. I have no idea how autotools works, so I copied it. What do we want?

Reply:
Yes, because we want pulse and alsa to be the default if available. But for pipewire, not yet.
Just drop
```
		AS_IF([test -z "$default_audio_method"],
			[default_audio_method=pipewire])
```

Reply:
ok, would people be able to build spd with pipewire support then?

Reply:
also, can't packagers who don't want that just disable it through flags or something? Either way, I did it, but now I dk how to build my own copy with pipewire support, must I pass something via flags or something? I never used autoconf, so have no idea

Reply:
> also, can't packagers who don't want that just disable it through flags or something?

Sure they can, but we as upstream do not want people to set it as default audio method yet, to avoid getting a flurry of bug reports, and rather only get reports from a few beta testers.

> now I dk how to build my own copy with pipewire support, must I pass something via flags or something?

You don't need anything, all audio methods are built by default, you just need to choose which one you want in speechd.conf.